### PR TITLE
Replace unlicensed fonts with system fonts

### PR DESCRIPTION
**Pull request by dishanest**
## Overview

On iOS, Begum and Lato are fonts that are not permitted for free commercial use for native apps. We replaced them with the available system fonts, seen here: 

[https://developer.apple.com/fonts/system-fonts/](https://developer.apple.com/fonts/system-fonts/)
[https://developer.apple.com/fonts/](https://developer.apple.com/fonts/)

## Changes Made

Replaced all instances of these fonts (and weights) with their respective system versions, after consultation w/ design:
- Begum -> New York
- Lato Regular -> Helvetica Regular
- Lato Bold -> Helvetica Neue Medium

Changed the app icon: 
- The Volume app icon uses the letter "V" from Helvetica, which isn't licensed for use on Android. 
- We changed it to a "V" from Garamond, which is the closest font licensed for both platforms. 

## Test Coverage

Checked that fonts display in runtime on all screens.

## Screenshots - Before

<details>

  <summary>Before - App Icon</summary>

Simulator Screen Shot - iPhone 13 Pro - 2022-03-14 at 17 18 02

</details>

<details>

  <summary>Before - Onboarding</summary>

Simulator Screen Shot - iPhone 13 Pro - 2022-03-14 at 17 18 16

</details>

<details>

  <summary>Before - HomeList</summary>

Simulator Screen Shot - iPhone 13 Pro - 2022-03-14 at 17 18 33

</details>

<details>

  <summary>Before - BrowserView</summary>

Simulator Screen Shot - iPhone 13 Pro - 2022-03-14 at 17 18 41

</details>

<details>

  <summary> Before - PublicationsList</summary>

Simulator Screen Shot - iPhone 13 Pro - 2022-03-14 at 17 18 46

</details>

<details>

  <summary> Before - PublicationDetail</summary>

Simulator Screen Shot - iPhone 13 Pro - 2022-03-14 at 17 18 50

</details>

<details>

  <summary> Before - SettingsView</summary>

Simulator Screen Shot - iPhone 13 Pro - 2022-03-14 at 17 19 00

</details>

## Screenshots - After

<details>

  <summary>After - App Icon</summary>

Simulator Screen Shot - iPhone 13 Pro - 2022-03-14 at 17 12 24

</details>

<details>

  <summary>After - Onboarding</summary>

Simulator Screen Shot - iPhone 13 Pro - 2022-03-14 at 17 13 42

</details>

<details>

  <summary>After - HomeList</summary>

Simulator Screen Shot - iPhone 13 Pro - 2022-03-14 at 17 13 56

</details>

<details>

  <summary>After - BrowserView</summary>

Simulator Screen Shot - iPhone 13 Pro - 2022-03-14 at 17 14 09

</details>

<details>

  <summary>After - PublicationsList</summary>

Simulator Screen Shot - iPhone 13 Pro - 2022-03-14 at 17 14 16

</details>

<details>

  <summary>After - PublicationDetail</summary>

Simulator Screen Shot - iPhone 13 Pro - 2022-03-14 at 17 14 26

</details>

<details>

  <summary>After - SettingsView</summary>

Simulator Screen Shot - iPhone 13 Pro - 2022-03-14 at 17 14 33

</details>

### CODE REVIEW

**Review by mathewscullin — APPROVED** [review]
Love to see it! 🎉

**Review by amyqxhuang — APPROVED** [review]
looks good!! 🤩🚀

just a note: for PRs where you have a lot of screenshots, it can be helpful to format them by putting them in an image tag like: `` where `image-url` is the url of the image you get after dragging it in. at least for me, my screen is not big enough to see the entire image 🥲

**Review by SylvanM — APPROVED** [review]
🎉

File: Volume/Views/AboutUsView.swift [hunk 1]
```diff
@@ -18,15 +18,15 @@ struct AboutUsView: View {
...
                     .padding([.bottom, .top])
                 
                 Header("The Team")
                     .padding(.top)
                 Text(Constants.teamInfo)
-                    .font(.latoRegular(size: 16))
+                    .font(.helveticaRegular(size: 16))
                     .lineSpacing(2)
                     .fixedSize(horizontal: false, vertical: true)
                     .padding([.bottom, .top])
```
Comment (Npikielny) [inline Volume/Views/AboutUsView.swift:32]: Sorry, I had to.
```suggestion
                    .padding(.vertical)
```

**Review by Npikielny — APPROVED** [review]
I want code 😤

**Review by NgaiJustin — APPROVED** [review]
🔥

**Review by vianxnguyen — APPROVED** [review]

**Pull request merged**